projects/rocprofiler-sdk/source/lib/python/rocpd/source/perfetto.cpp

```diff
@@ -177,27 +177,28 @@ write_perfetto(
     const tool::generator<types::memory_copies>&     memory_copy_gen,
     const tool::generator<types::scratch_memory>&    scratch_memory_gen,
     const tool::generator<types::memory_allocation>& memory_allocation_gen,
-    const tool::generator<types::counter>&           counter_collection_gen)
+    const tool::generator<types::counter>&           counter_collection_gen,
+    const tool::generator<types::pmc_event>&         pmc_event_gen)
 {
     namespace sdk    = ::rocprofiler::sdk;
     namespace common = ::rocprofiler::common;
 
-    static auto orig_process_track = ::perfetto::ProcessTrack::Current();
-    static auto orig_process_desc  = orig_process_track.Serialize();
+    static auto     orig_process_track = ::perfetto::ProcessTrack::Current();
+    static auto     orig_process_desc  = orig_process_track.Serialize();
+    static uint64_t global_flow_index  = 0;
+    static uint64_t global_track_index = 0;
 
     const auto&    tracing_session  = perfetto_session.tracing_session;
     const auto&    ocfg             = perfetto_session.config;
     const uint64_t this_pid         = process.pid;
     const uint64_t this_pid_init_ns = process.init;
-    const uint64_t this_nid         = process.nid;
     auto           command_line     = ::rocprofiler::sdk::parse::tokenize(process.command, " ");
 
-    auto uuid_pid       = common::fnv1a_hasher::combine(this_nid, this_pid_init_ns, this_pid);
-    auto this_pid_track = ::perfetto::Track{uuid_pid, ::perfetto::Track{}};
+    auto this_pid_track = ::perfetto::Track::Global(++global_track_index);
```

Comment:
```suggestion
    auto uuid_pid       = common::fnv1a_hasher::combine(this_nid, this_pid_init_ns, this_pid);
    auto this_pid_track = ::perfetto::Track{uuid_pid, ::perfetto::Track{}};
```
Can we go back to original?  If the objective is to just get "Sampling" tracks to create their own tracks, then just making a UUID unique should work?  I provided a suggestion in line 287.

Reply:
Done.